data/bloomberg

```diff
@@ -1,26 +1,48 @@
 # All .bloomberg domains
 bloomberg
 
+bbgevent.com
 bbhub.io
 bbthat.com
 bgov.com
-bloomberg.cn
+bloomberg.cn @cn
```

Comment:
`bloomberg.cn` doesn't have access point in mainland China.

Reply:
if I ping `bloomberg.cn` , I will get a US IP and it was blocked in mainland. (dns problem perhaps)

Reply:
> `bloomberg.cn` doesn't have access point in mainland China.

Done.